### PR TITLE
New CPU

### PR DESCRIPTION
**Pull request by polluks**

### CODE REVIEW

File: src/ca65/condasm.c [hunk 1]
```diff
@@ -460,6 +470,7 @@ int CheckConditionals (void)
         case TOK_IFP816:
         case TOK_IFPC02:
         case TOK_IFPSC02:
+        case TOK_IFPCE02:
```
Comment (greg-king5) [inline src/ca65/condasm.c]: The above two lines should be swapped (for sorting).

**Comment by greg-king5** [comment]
That new CPU (and its dot-keywords) should be described in several document files.

**Comment by polluks** [comment]
Well, I didn't release the docs because some addressing modes are not complete.

**Comment by oliverschmidt** [comment]
Just to make sure we don't have a communication deadlock here: This pull requst waits for completion of addressing modes and overall documentation.

Apart form that I propose to put the chrcvt aspect in a separate pull request.

**Comment by oliverschmidt** [comment]
There seems to be no momentum to separate this pull request so I went ahead and

- adjusted the Protovision Shop URL with https://github.com/cc65/cc65/commit/d2f012e4143de46e4f8892b90dc0dd2adeeaa45a
- took care of chrcvt with https://github.com/cc65/cc65/commit/8b685763d4c0845b6f877aa79ec6c6db890f7ffc

Regarding the the primary goal of the pull request - to introduce support for an additional CPU - I suggest to open a new pull request containing only the relevant changes. I presume this approach is the least effort for everybody.

**Comment by polluks** [comment]
Please take care these commits are independent
https://github.com/cc65/cc65/pull/266/commits/8bc4437effa58be07ccd4e91c920a4acfc225e83
https://github.com/cc65/cc65/pull/266/commits/cb120f1744e6f986a58ea6a9d6ba637f1815d558
https://github.com/cc65/cc65/pull/266/commits/55d65567b6241ad266750d723a40a588b892cc18
and these are only sim65 related
https://github.com/cc65/cc65/pull/266/commits/1ce398d83e1ab28c982b24dd4c03b60fcd231ba9
https://github.com/cc65/cc65/pull/266/commits/10373914ae6ff87597985c8fe87c0af6d0135a4e
https://github.com/cc65/cc65/pull/266/commits/c13bc3afaf3852a3a01ea907668e92a76591842a
sorry for the inconvenience.

**Comment by greg-king5** [comment]
The target platforms automatically choose a default CPU instruction-set.  Therefore, CPU directives shouldn't be in the include files.  Commit 55d65567b6241 must be changed.  It should remove the directive in "supervision.inc".

Currently, the Lynx target chooses the 65C02.  If it should choose the 65SC02, then "`src/common/target.c`" should be patched.

**Comment by polluks** [comment]
I see. I've done a commit https://github.com/polluks/cc65/commit/463c191427e5224fe0b499ea273241732fd33a8a but I don't know how to change https://github.com/cc65/cc65/commit/55d65567b6241ad266750d723a40a588b892cc18. A new PR contains every change.

**Pull request closed without merging**